docs/cheqd-cli/cheqd-cli-fee-abstraction.md

```diff
@@ -0,0 +1,241 @@
+# Leveraging Fee Abstraction through cheqd CLI
+
+## Overview
+
+A `cheqd-node` instance can be controlled and configured using the [cheqd Cosmos CLI](README.md).
+
+This document contains the Fee Abstraction commands for [paying for transactions](cheqd-cli-token-transactions.md) using governance-approved alternative tokens, e.g., a stablecoin such as USDC. The Fee Abstraction module routes requests for supported IBC denominations to [Osmosis DEX](https://app.osmosis.zone), and uses existing liquidity pools (e.g., the [CHEQ/USDC pool](https://app.osmosis.zone/pool/1273)) to convert the tokens on the fly. The underlying transaction on cheqd network is **always** funded in CHEQ tokens.
+
+Various commands are available for declaring fees in transactions, along with querying and configuring allowed denominations through governance proposals.
+
+## Supported IBC denominations
+
+Fees for transactions cannot be denominated in arbitrary alternative tokens, when attempting to pay for transactions with non-CHEQ tokens. Supported IBC denominations must be approved using decentralised governance.
+
+## Prerequisites for using Fee Abstraction
+
+The equivalent IBC denomination amount is required to pay for transactions. **Ensure this amount is available in the equivalent Osmosis account** related to the cheqd account/key provided in the transaction.
+
+### Looking up balance of equivalent Osmosis account
+
+You can find out if you've got sufficient balance in supported IBC denominations using the following methods:
+
+1. **Through Leap Wallet**
+   1. Ensure you have added the cheqd wallet you want to use for transactions to a supported desktop/mobile wallet. The recommended wallet app is [Leap Wallet](https://www.leapwallet.io/download). If you previously used Keplr Wallet, we recommend [migrating from Keplr Wallet to Leap Wallet](https://docs.cheqd.io/product/network/wallets/migrate) as it has better support for [looking up real-time gas prices](./cheqd-cli-token-transactions.md).
+   2. Once you've added the cheqd wallet account to Leap Wallet, use the network switcher to switch to **Osmosis**. This will allow you to see the balances you have on Osmosis chain, including native OSMO as well as any IBC denominations such as USDC.
+2. **Converting your cheqd address to Osmosis using CLI**
+   1. TASOS: We should probably add this under the [key management](./cheqd-cli-key-management.md) and link to it from here.
```

Comment:
I think you know this technique, it's basically running `cheqd-noded keys parse` first on a wallet address, and then using it again on the output of that command with the osmo prefix provided to derive the osmo address.

Reply:
That's only available converting from an external address to the chain-native prefixed version, e.g. osmo to cheqd. Not vice-versa.

That's part of the toolset I wanted to add at a later time, without requiring the Osmosis CLI to do the conversion (since that's how the above key parsing routine works).